### PR TITLE
feat: Wholesale results sort

### PR DESCRIPTION
**Pull request by johevemi**
# Description

## Pull-request quality

- [ ] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/settlement_report/settlement_report_job/domain/csv_column_names.py [hunk 1]
```diff
@@ -2,7 +2,7 @@ class CsvColumnNames:
     energy_business_process = "ENERGYBUSINESSPROCESS"
     energy_quantity = "ENERGYQUANTITY"
     energy_supplier_id = "ENERGYSUPPLIERID"
-    grid_area_code = "METERINGGRIDAREAID"
+    metering_grid_area = "METERINGGRIDAREAID"
```
Comment (BjarkeMeier) [inline source/databricks/settlement_report/settlement_report_job/domain/csv_column_names.py:5]: Isn't it a grid area code?

Comment (BjarkeMeier) [reply source/databricks/settlement_report/settlement_report_job/domain/csv_column_names.py:5]: How about using the usual names in the domain and only use the CSV column names in the values?
I'd also prefer `type_of_mp` to be renamed to `metering_point_type`.

Comment (johevemi) [reply source/databricks/settlement_report/settlement_report_job/domain/csv_column_names.py:5]: it is the name of the csv column in settlement report, i changed it once to this but someone changed it back

Comment (johevemi) [reply source/databricks/settlement_report/settlement_report_job/domain/csv_column_names.py:5]: it looks like I just forgot to change it in a former PR

Comment (johevemi) [reply source/databricks/settlement_report/settlement_report_job/domain/csv_column_names.py:5]: what you suggest is how it was before, lets talk about it at tech refin

File: source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py [hunk 2]
```diff
@@ -68,4 +68,13 @@ def prepare_for_csv(
         ),
     ]
 
-    return wholesale.select(select_columns)
+    return wholesale.select(select_columns).orderBy(
+        F.col(CsvColumnNames.grid_area_code),
+        F.col(CsvColumnNames.energy_supplier_id),
```
Comment (JesperGomes) [inline source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py:73]: Which user is this intended for? energy_supplier_id is not relevant for the energy supplier.

Comment (johevemi) [reply source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py:71]: every user, this is correct according to confluence doc

Comment (JesperGomes) [reply source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py:71]: oh, yes - sorry

File: source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py [hunk 3]
```diff
@@ -66,4 +66,13 @@ def prepare_for_csv(
         ),
     ]
 
-    return wholesale.select(select_columns)
+    return wholesale.select(select_columns).orderBy(
```
Comment (JesperGomes) [inline source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py:69]: unit test? 
I have a PR that does it for other factories. See for example here:
https://github.com/Energinet-DataHub/opengeh-wholesale/blob/39904fc2b0243aaecbdad4bc76f9384b15aea668/source/databricks/settlement_report/tests/domain/energy_results/test_energy_prepare_for_csv.py#L73

Comment (johevemi) [reply source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py:69]: I will create a test for this

Comment (johevemi) [reply source/databricks/settlement_report/settlement_report_job/domain/wholesale_results/prepare_for_csv.py:69]: done

File: source/databricks/settlement_report/tests/domain/wholesale_results/test_wholesale_prepare_for_csv.py [hunk 4]
```diff
@@ -234,3 +238,59 @@ def test_mapping_of_settlement_method(
 
     # Assert
     assert actual.collect()[0]["SETTLEMENTMETHOD"] == expected_settlement_method
+
+
+def test_when_multiple_rows__returns_dataframe_with_expected_ordering(
+    spark: SparkSession,
+) -> None:
+    # Arrange
+    spec = [
```
Comment (JesperGomes) [inline source/databricks/settlement_report/tests/domain/wholesale_results/test_wholesale_prepare_for_csv.py:247]: you only have three rows (with different grid areas). So, this test only verifies that you grid area is ordered as expected!?

**Pull request merged**